packages/grid/x-data-grid-pro/src/tests/editComponents.DataGridPro.test.tsx

```diff
@@ -318,7 +324,9 @@ describe('<DataGridPro /> - Edit Components', () => {
         />,
       );
       const cell = getCell(0, 0);
-      cell.focus();
+      fireEvent.mouseUp(cell);
+      fireEvent.click(cell);
```

Comment:
why not `fireClickEvent(cell)`?

Reply:
Maybe we could use `mousePress` utility method from the core

Reply:
`mousePress` doesn't work. Some tests fail but I don't know why.